internal/data/organizations.go

```diff
@@ -27,9 +27,12 @@ const (
 )
 
 type Organization struct {
-	ID                             string `json:"id" db:"id"`
-	Name                           string `json:"name" db:"name"`
-	TimezoneUTCOffset              string `json:"timezone_utc_offset" db:"timezone_utc_offset"`
+	ID                string `json:"id" db:"id"`
+	Name              string `json:"name" db:"name"`
+	TimezoneUTCOffset string `json:"timezone_utc_offset" db:"timezone_utc_offset"`
+	// SMSResendInterval is the time period that SDP will wait to resend the invitation SMS to the receivers that aren't registered.
+	// If it's nil means resending the invitation SMS is deactivated.
+	SMSResendInterval              *int64 `json:"sms_resend_interval" db:"sms_resend_interval"`
```

Comment:
Thank you for the comment, @marwen-abid. In this case, the [`0` deactivates the feature](https://github.com/stellar/stellar-disbursement-platform-backend/pull/65/files#diff-1a7b591c65aba9e95d86d93844749ec82bf92d33d8c5a75d511e7c7a718dd5e3R203-R209). The `nil` means we don't want to change the current value, otherwise, the frontend would send this attribute in every request. WDYT?

Reply:
That sounds good to me! Thank you for clarifying it